5.1.0

```diff
@@ -0,0 +1,4 @@
+Collecting isort
+  Downloading isort-5.6.4-py3-none-any.whl (98 kB)
+Installing collected packages: isort
+Successfully installed isort-5.6.4
```

Comment:
Hi there. Please check "Has patch" on the ticket to get in the review queue. Also please mind what you check into the repo; this file shouldn't be here. Cheers.

Reply:
Now the file is back...

Reply:
I'm sorry how would I remove this. Do I just delete it on Github?

Reply:
No problem! That would be one way. Whatever platform you're using to stage your changes should also give you control over which files to avoid staging.

Reply:
Ok I was able to remove the file and push the updated content. I just recently learned to use git reset to unstage it. Thanks for your help.